packages/widget-chat/src/components/activity-title-bar/index.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {findRenderedDOMComponentWithClass, renderIntoDocument} from 'react-addons-test-utils';
+
+import ActivityTitleBar from '.';
+
+let user;
+let component;
+
+describe(`ActivityTitleBar component`, () => {
```

Comment:
Take it up with the web client team :)
(my email response didn't put it on the right thread!)